### PR TITLE
Add docs build script

### PR DESCRIPTION
**Pull request by eatonphil**
Current build size is 500kb or so. We should work on that.

### CODE REVIEW

File: docs/webpack.config.prod.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  new webpack.DefinePlugin({
+    'process.env': {
+      'NODE_ENV': JSON.stringify('production')
+    },
+    'ENV_DEVTOOLS_DISABLED': JSON.stringify(process.env.DEVTOOLS_DISBLED),
+    'ENV_APP_ROOT': JSON.stringify(process.env.APP_ROOT),
+    'ENV_API_ROOT': JSON.stringify(process.env.API_ROOT),
+    'ENV_LOGIN_ROOT': JSON.stringify(process.env.LOGIN_ROOT),
+    'ENV_GA_ID': JSON.stringify(process.env.GA_ID),
+    'ENV_VERSION': JSON.stringify(_package.version)
```
Comment (eatonphil) [inline docs/webpack.config.prod.js]: Need to drop these

File: docs/package.json [hunk 2]
```diff
@@ -4,6 +4,8 @@
   "description": "Linode API Docs",
   "main": "src/index.js",
   "scripts": {
+    "build:webpack": "cross-env NODE_ENV=production webpack --config webpack.config.prod.js",
+    "build": "npm run clean && npm run build:webpack",
```
Comment (mparke) [inline docs/package.json]: does `clean` exist?

Comment (eatonphil) [inline docs/package.json]: whoops, will add. Thank you

File: docs/package.json [hunk 3]
```diff
@@ -39,6 +42,7 @@
     "phantomjs-polyfill": "0.0.2",
     "phantomjs-prebuilt": "^2.1.7",
     "react-addons-test-utils": "^15.1.0",
+    "rimraf": "^2.4.3",
```
Comment (mparke) [inline docs/package.json]: why vs. `rm -rf` ?

Comment (eatonphil) [inline docs/package.json]: Heh, I have no idea. It's been in there.

Comment (mparke) [inline docs/package.json]: use `rm -rf` explicitly until we need another dep?

**Pull request merged**